src/code_80057C60.c

```diff
@@ -310,11 +310,18 @@ s32 gObjectParticle1[gObjectParticle2_SIZE];
 Collision D_8018C3B0;
 /**
  * List of object list indices used for:
- *   Bats in Banshee's Boardwalk (but only 1 player mode?)
+ * - Bats in Banshee's Boardwalk (but only 1 player mode?)
+ * - Train 0 smoke in Kalimari Desert
+ * - Ferry 0 smoke in DK Jungle
```

Comment:
This is good. But what do you mean by the 0? Is that the index value?

Reply:
yeah it's the index